packages/shared/src/libs/helpers/configure-query-string/configure-query-string.helper.ts

```diff
@@ -0,0 +1,22 @@
+import { EMPTY_LENGTH } from "../../constants/constants.js";
+
+const configureQueryString = <T extends Record<string, string>>(
```

Comment:
we need to use url search params instead, also I wonder why we use a different approach here from pagination hook

Reply:
I made the helper use the same approach as pagination hook.

Reply:
looks a bit complicated, we have getUrl method inside base-http-api, so I believe this helper should have the same logic. If so, we also will need to update usePagination hook, and  base-http-api to use this helper instead of their implementation

Reply:
I updated the helper to use the same logic as was in getUrl and updated base-http-api. But I cannot use this helper in usePagination because it doesn't create url with query, it just adds its parameters to existing. So it needs another helper which may be redundant or the configureQueryString will look more complicated.